<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgCalibDialog</class>
 <widget class="QDialog" name="ImgCalibDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
border:none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="titlebar" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget
{
background-color:#282f35;
border:1px solid #4a525a ;
border-radius:2px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="title">
            <property name="styleSheet">
             <string notr="true">QLabel
{
background-color: transparent;
font: 18px &quot;Microsoft YaHei&quot;;
color: #AAB3C1;
margin-left:20px;
border:0px;
}

QLabel:hover{
background-color: transparent;
border:0px;
}</string>
            </property>
            <property name="text">
             <string>校准</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_btn">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background:transparent;
padding:4px;
margin:-1px;
border-style:solid;
border-radius:1px;
border-color:#777;
border-width:1px;
image:url(:/2D/2DWidget/Resource/close.png);}
QPushButton:hover{
background:#dd0000;
padding:0px;
border-style:solid;
border-radius:1px;
border-width:1px;
border-color:#ddd;
color:white;}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
background: #14191C;
border:0px solid transparent;
}
</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="lengthLab">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel
{
background-color: transparent;
font: 14px &quot;Microsoft YaHei&quot;;
color: #AAB3C1;
border:0px;
}</string>
                </property>
                <property name="text">
                 <string>长度</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lengthEdit">
                <property name="minimumSize">
                 <size>
                  <width>210</width>
                  <height>34</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>210</width>
                  <height>34</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit
{
border: 1px solid #4a525a;
border-radius:0px;
background:#282f35;
color:#e1e8f1;
font:normal 12px  Microsoft YaHei;
}
QLineEdit:focus
{
border:3px solid #9dadc1;
border-radius:0px
}</string>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhNone</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>20</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="cancelBtn">
                <property name="minimumSize">
                 <size>
                  <width>118</width>
                  <height>42</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>118</width>
                  <height>42</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69,80,92,1),stop: 1.0 rgba(58,71,86,1));
border:0px solid #4a525a;
border-radius:0px;
color:#FFFFFF;
font: normal 16px  Microsoft YaHei;
}


QPushButton:pressed{
background-color:#14191C;
border:1px solid #404B56;
}

QPushButton:disabled{
opacity:0.39;
}</string>
                </property>
                <property name="text">
                 <string>取消</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="okBtn">
                <property name="minimumSize">
                 <size>
                  <width>118</width>
                  <height>42</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>118</width>
                  <height>42</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(60,140,227,1),stop: 1.0 rgba(32,108,195,1));
border:1px solid rgb(74, 82, 90);
border-radius:0px;
color:#FFFFFF;
font: normal 16px  Microsoft YaHei;
}

QPushButton:hover{
color:#ffffff;
}

QPushButton:pressed
{
background-color:#14191C;
color: #FFFFFF;
}

QPushButton:disabled
{
border:1px solid rgb(74, 82, 90, 50);
color:rgb(189, 205, 221, 50);
opacity:0.39;
}</string>
                </property>
                <property name="text">
                 <string>确定</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../readingprocess.qrc"/>
 </resources>
 <connections/>
</ui>
